Libraries/plist/Tools/PlistBuddy.cpp

```diff
@@ -499,7 +499,14 @@ Merge(plist::Object *object, std::string const &mergeSource, std::queue<std::str
 
     if (auto targetDictionary = plist::CastTo<plist::Dictionary>(target)) {
         if (auto mergeObjectDictionary = plist::CastTo<plist::Dictionary>(mergeObject.get())) {
-            targetDictionary->merge(mergeObjectDictionary);
+            // Check to see if duplicate keys exists between target and merge dictionary,
+            // and print warning that those keys will be skipped.
+            for (auto const &mergeKey : *mergeObjectDictionary) {
+                if (targetDictionary->value(mergeKey)) {
+                    fprintf(stderr, "Skipping duplicate key: %s\n", mergeKey.c_str());
```

Comment:
This for block is only there to print diagnostic about what keys are going to be skipped on
  targetDictionary->merge(mergeObjectDictionary, false);
on line 509.
If you wan't I can have this block removed, and not print this message.